crates/abq_cli/src/main.rs

```diff
@@ -498,10 +515,27 @@ async fn abq_main() -> anyhow::Result<ExitCode> {
                 tls_cert: resolved_tls,
                 rwx_access_token_kind: _resolved_rwx_access_token_kind,
                 queue_location,
-            } = resolve_config(token, queue_addr, tls_cert, None, &access_token, &run_id).await?;
+            } = resolve_config(
+                token,
+                queue_addr,
+                tls_cert,
+                None,
+                &access_token,
+                &run_id,
+                explicit_run_id_provided,
+            )
+            .await?;
 
             let client_auth = resolved_token.into();
 
+            if let QueueLocation::Unsupported(error_message) = &queue_location {
+                let mut cmd = Cli::command();
+                Err(cmd.error(
+                    ErrorKind::InvalidValue,
+                    format!("`abq report` failed to run. {}", error_message),
```

Comment:
I'd suggest we change the error message as above. Is it possible for us to pull out this and the config resolution into one function both `test` and `report` use?